p2p/tests/integration/p2p.rs

```diff
@@ -269,7 +269,7 @@ async fn start_network(
         broker: broker.clone(),
         messages,
     };
-    drop(actor.start().await);
+    let _actor_addr = actor.start().await;
```

Comment:
This is an important error message getting lost. Should have some feedback.

Reply:
Gave wrong impression in PR description: `actor.start().await` always return `Addr` not `Result`.
So we can safely just write:
```rust
actor.start().await;
```